test/intl_suggestors/intl_migrator_test.dart

```diff
@@ -767,11 +874,11 @@ void main() {
         );
 
         String expectedFileContent1 =
-            "\n  static String foo(String displayName) => Intl.message('Create one from any \${displayName} by selecting Save As Template', args: [displayName], name: 'TestClassIntl_foo');";
+            "\n  static String createOneFromAnyBy(String displayName) => Intl.message('Create one from any \${displayName} by selecting Save As Template', args: [displayName], name: 'TestClassIntl_createOneFromAnyBy');";
         String expectedFileContent2 =
-            "\n  static String bar(String displayName) => Intl.message('Create one from any \${displayName} by selecting Save As Template', args: [displayName], name: 'TestClassIntl_bar');";
+            "\n  static String createOneFromAnyBy1(String displayName) => Intl.message('Create one from any \${displayName} by selecting Save As Template', args: [displayName], name: 'TestClassIntl_createOneFromAnyBy1');";
```

Comment:
This is more a thought that either comes from misunderstanding or probably would be work for another task, but these two message strings look identical so it feels like there shouldn't be a `createOneFromAnyBy1` method and should instead be just one. I don't think this is blocking or anything just something I noticed.

Reply:
This test case is intentional to show method names for similar strings, In current PR we are not reusing the existing function via codemod.
I think this is being handled in a different branch, to reuse the existing and remove the unused functions.
We are taking the first 5 words from the strings and creating an INTL function and if any function name matches with the existing created we are appending and incrementing the 1,2,3... suffix.

Reply:
Ok sounds good to me, thanks for clarifying.